Cliqz/Privacy/UI/ControlCenter/OverviewViewController.swift

```diff
@@ -77,18 +81,19 @@ class BlockedRequestsView: UIView {
 	}
 	
 	func setStyles() {
-		countView.textColor = UIColor.cliqzBluePrimary
-		titleView.textColor = UIColor.cliqzBluePrimary
-		switchControl.onTintColor = UIColor.cliqzBlueTwoSecondary
-		switchControl.thumbTintColor = UIColor.cliqzURLBarColor
+		titleLabel.textColor = UIColor.cliqzBluePrimary
+		titleLabel.font = UIFont.boldSystemFont(ofSize: 18)
+		switchControl.onTintColor = UIColor.cliqzBluePrimary
 		switchControl.transform = CGAffineTransform(scaleX: 0.75, y: 0.75)
         switchControl.addTarget(self, action: #selector(switchValueChanged), for: .valueChanged)
 		notchView.image = UIImage(named:"notch")
-		descriptionLabel.textColor = ControlCenterUI.textGray
+		descriptionLabel.textColor = UIColor(colorString: "97A4AE")
```

Comment:
can you move this color to `ControlCenterUI`

Reply:
I don't know how to name it :)

Reply:
it is gray as well, right?

Reply:
Yes

Reply:
what about `descriptionGray`